backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -226,4 +227,17 @@ class CardMutationService {
         }
         return true
     }
+
+    @GraphQLDescription("Sends a confirmation mail to the user when the card creation was successful")
+    fun sendCardCreationConfirmationMail(dfe: DataFetchingEnvironment, project: String, recipientAddress: String, recipientName: String, deepLink: String): Boolean {
+        val context = dfe.getContext<GraphQLContext>()
+        val jwtPayload = context.enforceSignedIn()
+        transaction { AdministratorEntity.findById(jwtPayload.adminId) ?: throw UnauthorizedException() }
+        val backendConfig = dfe.getContext<GraphQLContext>().backendConfiguration
+        val projectConfig =
+            context.backendConfiguration.projects.find { it.id == project }
+                ?: throw ProjectNotFoundException(project)
+        Mailer.sendCardCreationConfirmationMail(backendConfig, projectConfig, deepLink, recipientAddress, recipientName)
+        return true
```

Comment:
as far as i read void return type goes against graphQL best practises and you would have to define a void scalar. 
If i remove boolean return type i can not update the graphql schema.
https://stackoverflow.com/a/61714123
Dunno if the true lleads to a status code tbh

Reply:
Yeah, we used to return a Boolean flag (which I believe was usually always `true` (never `false`) as nothing or `null` is returned in the error case).
Maybe that was not the best design decision from an API evolution standpoint; maybe it is better to always define an "object" type for every query, so that you can enrich the result of the API endpoint with other data without having to define a whole new query (and deprecating the old one). (that is why we have some graphql endpoints named `someQueryV2`).

Reply:
if we want to change this you can create a separate issue for this

Reply:
Yep, but for the administration endpoints, it's not really relevant, as we can break backwards compatibility for those...